src/kuery.ts

```diff
@@ -0,0 +1,80 @@
+import { drop, flow, keys, map, orderBy, take } from 'lodash/fp';
+import QueryCompiler from './compiler';
+import { Query } from './types';
+
+export interface KueryOptions {
+  skip: number;
+  limit: number;
+  sort: Record<string, number>;
+}
+
+
+class Kuery<T extends object = Record<string, any>> {
```

Comment:
Don't add duplicate newlines.

```suggestion
}

class Kuery<T extends object = Record<string, any>> {
```